changelog.d/33.feature.md

```diff
@@ -0,0 +1,34 @@
+* Support multiple algorithms and keys
+
+  Existing code made key rollovers or algorithm changes hard and
+  basically required a breaking change: Once any of `JWT_ALGORITHM`,
+  `JWT_SECERT_KEY`, or `JWT_PRIVATE_KEY`/`JWT_PUBLIC_KEY` were
```

Comment:
There's a type in `JWT_SECERT_KEY`, and in two spots just below.

Reply:
well spotted, thank you!